harness/determined/pytorch/_pytorch_context.py

```diff
@@ -303,7 +363,9 @@ def set_profiler(self, *args: List[str], **kwargs: Any) -> None:
         when training.
         """
         self.profiler = torch.profiler.profile(
-            on_trace_ready=torch.profiler.tensorboard_trace_handler(str(get_base_path({}))),
+            on_trace_ready=torch.profiler.tensorboard_trace_handler(
+                str(tensorboard.get_base_path({}))
```

Comment:
prolly a bug that this uses `tensorboard.get_base_path()` rather than `self.get_tensorboard_path()`.

Though I know `get_tensorboard_path` wasn't an option when you originally wrote this.

Reply:
fixed